src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperBlockBuilder.cs

```diff
@@ -35,17 +35,20 @@ public TagHelperBlockBuilder(TagHelperBlock original)
         /// <param name="tagName">An HTML tag name.</param>
         /// <param name="start">Starting location of the <see cref="TagHelperBlock"/>.</param>
         /// <param name="attributes">Attributes of the <see cref="TagHelperBlock"/>.</param>
+        /// <param name="selfClosing">The <c>bool</c> indicating whether or not the current tag is self-closing.</param>
```

Comment:
- use the same doc for this parameter as in `TagHelperExecutionContext`
- also, match parameter order and property comments as well